### ISSUE TITLE
Setting for journal max size

### ISSUE THREAD

**Issue by dsm1212** [openmediavault/openmediavault#1457]
I'm running omv6. My journal file seems to grow and grow with the default debian config. Would be nice to have a way in the OMV ui to set tha max journal size.  This is SystemMaxUse in journald.conf. That conf file has other settings too you might want to allow. I changed this myself, but I'm surprised folks with tiny installs are not raising this issue.

**Comment by T-6** [comment]
Did you check why the journal is growing to such an extent? Maybe there is an underlaying issue that needs to be resolved.

**Comment by dsm1212** [comment]
I don't think this accumulates all that much but since it is omv related I do see this set of message every 15 mins. Any idea what this is?

steve

Jan 15 11:15:01 warehouse21 omv-mkrrdgraph[3039394]: Failed to build graph (plugin=disk, period=hour): [Errno 2] No such file or directory: '/var/lib/openmediavault/rrd/disk-octets-disk/by-uuid/32f0aa31-3240-4e9d-a527-7fdd14ff140f-hour.png'
Jan 15 11:15:03 warehouse21 omv-mkrrdgraph[3039394]: Failed to build graph (plugin=disk, period=day): [Errno 2] No such file or directory: '/var/lib/openmediavault/rrd/disk-octets-disk/by-uuid/32f0aa31-3240-4e9d-a527-7fdd14ff140f-day.png'
Jan 15 11:15:05 warehouse21 omv-mkrrdgraph[3039394]: Failed to build graph (plugin=disk, period=week): [Errno 2] No such file or directory: '/var/lib/openmediavault/rrd/disk-octets-disk/by-uuid/32f0aa31-3240-4e9d-a527-7fdd14ff140f-week.png'
Jan 15 11:15:07 warehouse21 omv-mkrrdgraph[3039394]: Failed to build graph (plugin=disk, period=month): [Errno 2] No such file or directory: '/var/lib/openmediavault/rrd/disk-octets-disk/by-uuid/32f0aa31-3240-4e9d-a527-7fdd14ff140f-month.png'
Jan 15 11:15:09 warehouse21 omv-mkrrdgraph[3039394]: Failed to build graph (plugin=disk, period=year): [Errno 2] No such file or directory: '/var/lib/openmediavault/rrd/disk-octets-disk/by-uuid/32f0aa31-3240-4e9d-a527-7fdd14ff140f-year.png'

**Comment by dsm1212** [comment]
I had a mirror of my boot disk fail and I replaced it a month or two ago. I wonder if some remnant of that didn't get cleaned up in the OMV config?

steve

**Comment by dsm1212** [comment]
I notice there are no performance stats charts for my large array (ext4 hardware raid). Probably these messages are somehow due to that.  At some point when I booted my system this array changed from sda to sdb. It was either after I did the disk replacement or maybe it was after I resized one of the partitions on the boot disk. Maybe is somehow due to one of these changes but everything else is working fine so maybe it is something just in the RRD config?

steve

**Comment by dsm1212** [comment]
Must have been an rrd change. Fixed these messages with:

sudo mkdir -p /var/lib/openmediavault/rrd/disk-octets-disk/by-uuid
sudo mkdir -p /var/lib/openmediavault/rrd/disk-ops-disk/by-uuid
sudo mkdir -p /var/lib/openmediavault/rrd/disk-time-disk/by-uuid

It looks like it used to create files in the rrd folder and now it uses subfolders which no one created??

steve

**Comment by votdev** [comment]
IMO these directories should be automatically created by `collectd`.

**Comment by dsm1212** [comment]
Interesting, while looking at this I also noticed I had no diskio graphs for sdb. sdb appeared I think after I modified the partitions on sda. In collectd.conf.d/disk.conf it still listed sda and no sdb. It seems like I missed a conversion or update for collectd that left a few things out of date. I don't remember having any errors, but maybe one of the package updates botched it or there is an error case here when the boot disk changes that wasn't tested by the collectd team.
